src/compressed_tensors/quantization/utils/helpers.py

```diff
@@ -167,7 +169,7 @@ def infer_quantization_status(model: Module) -> Optional["QuantizationStatus"]:
     return None
 
 
-def is_module_quantized(module: Module) -> bool:
+def is_submodule_quantized(module: Module) -> bool:
```

Comment:
why a name change here to submodule? a `Module` is the type passed in, afaik there is no notion of a `torch.nn.Submodule`

Reply:
I also do not agree with the name change for the above reason. While I understand if the intention was to make it clear that only child modules are quantized, it doesn't change the fact that calling `is_module_quantized` on a non-child module will return False, which is correct.

Reply:
ok no problem

Reply:
pasting slack convo here:
Brian:
> wondering what the motivation is for a name change of a public function

George:
> Motivation is that quantized layers are always submodules (or child modules)

Reply:
Yeah thanks makes sense!

Reply:
I think in that case it's worth indicating in the doc string what will happen for modules vs child modules, strong aversion to breaking name changes though.

Reply:
I didn't put details on parent vs child docs, but added info on what type of layers should return True.